install-devenv.sh

```diff
@@ -36,13 +42,17 @@ curl -fsSL \
 
 echo "${sha256}  ${tmpd}/${archive}" | /usr/bin/shasum -a 256 --check --status
 
+# Question: What is the --strip-components doing here?
 tar --strip-components=1 -C "$devenv_python_root" -x -f "${tmpd}/${archive}"
 
+# Question: Is there a reason to use the SSH URL here?
```

Comment:
yes, by making sure we clone over SSH we exercise that engineers have their github ssh keys set up correctly

i'll be coming up with a separate github ssh key setup thing soon